cub/util_debug.cuh

```diff
@@ -51,6 +51,79 @@ CUB_NAMESPACE_BEGIN
  */
 
 
+/**
+ * `CUB_DETAIL_DEBUG_LEVEL_*`: Implementation details, internal use only:
```

Comment:
This doesn't need to be a doxygen comment, but we should provide them for the user-visible macros. See cub/detail/detect_cuda_runtime.cuh for an example of this.